modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/Question.es.js

```diff
@@ -384,6 +384,7 @@ export default withRouter(
 											<Answer
 												answer={answer}
 												answerChange={answerChange}
+												canMarkAsAnswer={question.actions.replace?true:false}
```

Comment:
I am not sure is Source Formatter will accept that or change it automatically. Usually, the format for this is something like:
```
canMarkAsAnswer={
      question.actions.replace
              ? true
              : false
}
```